packages/web/src/components/track/LockedTrackDetailsTile.tsx

```diff
@@ -50,21 +59,28 @@ export const LockedTrackDetailsTile = ({
   )
 
   const dogEarType = getDogEarType({
-    streamConditions
+    streamConditions,
+    downloadConditions
   })
   const label = `${title} by ${owner.name}`
   const isCollectibleGated = isContentCollectibleGated(streamConditions)
   const isUSDCPurchaseGated = isContentUSDCPurchaseGated(streamConditions)
 
-  let IconComponent = IconSpecialAccess
-  let message = messages.specialAccess
+  let IconComponent: Nullable<IconComponent>
```

Comment:
awesome. just curious - how does `Nullable<IconComponent>` allow the var to be `undefined`? i thought null and undefined were separate.

Reply:
you can declare a variable but not assign it and that's valid syntax. if you never assign it, it would yell at you